prow/hook/server.go

```diff
@@ -165,16 +167,7 @@ func (s *Server) needDemux(eventType, srcRepo string) []plugins.ExternalPlugin {
 
 	for repo, plugins := range s.Plugins.Config().ExternalPlugins {
 		// Make sure the repositories match
-		var matchesRepo bool
-		if repo == srcRepo {
-			matchesRepo = true
-		}
-		// If repo is an org, we need to compare orgs.
-		if !matchesRepo && !strings.Contains(repo, "/") && repo == srcOrg {
-			matchesRepo = true
-		}
-		// No need to continue if the repos don't match.
-		if !matchesRepo {
+		if repo != srcRepo && repo != srcOrg {
```

Comment:
Seems orthogonal?

Reply:
Yeah it totally is. I was irritated by the unnecessary complexity and figured it was a small enough clean up to include in this PR.

/hold

Reply:
oops
/hold cancel